### PR TITLE
OAS-6824 | Remove error stream support, improve log output

### PR DESCRIPTION
**Pull request by nikita-vanyasin**
Error stream currently is not used and actually is not desired for reverse-proxy scenario

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -204,14 +205,14 @@ func (c *portForwardClientImpl) CopyToStream(ctx context.Context, stream httpstr
 	errCh := make(chan error, 2)
 	// Copy from the forward port connection to the client stream
 	go func() {
-		c.log.Debug().Msgf("PortForward copying data to the client stream")
+		c.log.Debug().Msgf("PortForward copying data from stream %d to client connected at %s", stream.Identifier(), addr)
```
Comment (Robert-Stam) [inline client/client.go]: Please use structured logging, so Msgf --> Msg and configurable data in 
.Int("identifier", stream.Identifier()).Str("address", add)

File: client/client.go [hunk 2]
```diff
@@ -204,14 +205,14 @@ func (c *portForwardClientImpl) CopyToStream(ctx context.Context, stream httpstr
...
 		_, err := io.Copy(stream, conn)
 		errCh <- err
 	}()
 
 	// Copy from the client stream to the port connection
 	go func() {
-		c.log.Debug().Msg("PortForward copying data from client stream to forward port")
+		c.log.Debug().Msgf("PortForward copying data from client connection (%s) to stream %d", addr, stream.Identifier())
```
Comment (Robert-Stam) [inline client/client.go]: Please use structured logging, so Msgf --> Msg and configurable data in 
.Int("identifier", stream.Identifier()).Str("address", add)

File: client/http_stream_handler_test.go [hunk 3]
```diff
@@ -101,87 +93,74 @@ func (*fakeConn) CloseChan() <-chan bool                                      {
...
-	errorStream := newFakeHTTPStream()
-	errorStream.headers.Set(api.HeaderStreamType, api.StreamTypeError)
-	complete, err = p.add(errorStream)
-	if err != nil {
-		t.Fatalf("unexpected error adding error stream to pair: %v", err)
+		t.Fatalf("unexpected error adding data stream: %v", err)
 	}
 	if !complete {
-		t.Fatal("unexpected incomplete")
+		t.Fatalf("expected complete=true")
```
Comment (Robert-Stam) [inline client/http_stream_handler_test.go]: Fatalf --> Fatal

File: client/http_stream_handler_test.go [hunk 4]
```diff
@@ -191,15 +170,15 @@ func TestGetStreamPair(t *testing.T) {
...
-	// make sure monitorStreamPair completed
+	// make sure monitorStream completed
 	<-monitorDone
-	if h.hasStreamPair("2") {
-		t.Fatal("expected stream pair to be removed")
+	if h.hasStream("2") {
+		t.Fatal("expected stream to be removed")
 	}
 	if !conn.removeStreamsCalled {
 		t.Fatalf("connection remove stream not called")
```
Comment (Robert-Stam) [inline client/http_stream_handler_test.go]: Fatalf --> Fatal

**Review by Robert-Stam — APPROVED** [review]
Some small remarks, further LGTM

File: client/client.go (hunk 1 above)
Comment (nikita-vanyasin) [reply client/client.go]: done 👍🏻

File: client/client.go (hunk 2 above)
Comment (nikita-vanyasin) [reply client/client.go]: done 👍🏻

File: client/http_stream_handler_test.go (hunk 3 above)
Comment (nikita-vanyasin) [reply client/http_stream_handler_test.go]: done 👍🏻

**Pull request merged**